analytics-integrations/moengage/src/test/java/com/segment/analytics/internal/integrations/MoEngageTest.java

```diff
@@ -0,0 +1,141 @@
+package com.segment.analytics.internal.integrations;
+
+import android.app.Activity;
+import android.app.Application;
+import android.content.Context;
+import android.os.Bundle;
+
+import com.moe.pushlibrary.MoEHelper;
+import com.segment.analytics.Analytics;
+import com.segment.analytics.IntegrationTestRule;
+import com.segment.analytics.TestUtils;
+import com.segment.analytics.Traits;
+import com.segment.analytics.core.tests.BuildConfig;
+import com.segment.analytics.internal.model.payloads.util.AliasPayloadBuilder;
+import com.segment.analytics.internal.model.payloads.util.GroupPayloadBuilder;
+import com.segment.analytics.internal.model.payloads.util.IdentifyPayloadBuilder;
+import com.segment.analytics.internal.model.payloads.util.ScreenPayloadBuilder;
+import com.segment.analytics.internal.model.payloads.util.TrackPayloadBuilder;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.verifyNoMoreInteractions;
+
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class, emulateSdk = 18, manifest = Config.NONE)
+@PowerMockIgnore({ "org.mockito.*", "org.robolectric.*", "android.*", "org.json.*"  })
+@PrepareForTest(MoEngageIntegration.class)
+public class MoEngageTest {
+
+    @Rule public PowerMockRule rule = new PowerMockRule();
+    @Rule public IntegrationTestRule integrationTestRule = new IntegrationTestRule();
+
+    @Mock Application application;
+    @Mock Context context;
+    @Mock Analytics analytics;
+    MoEngageIntegration integration;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        integration = new MoEngageIntegration();
+        integration.mHelper = MoEHelper.getInstance(application);
+    }
+
+    @Test public void initialize(){
+        //nothing to be done
```

Comment:
Verify that `MoEHelper.getInstance(context)` was called and `mHelper` is not null.